modules/apps/jaggery/mdm/src/mdm/config/config.json

```diff
@@ -5,6 +5,10 @@
     "httpURL" : "%http.ip%",
     "iOSConfigRoot" : "%https.ip%/ios-enrollment/",
     "iOSAPIRoot" : "%https.ip%/ios/",
+	"carbonServer":"%https.ip%",
```

Comment:
I have added a separate entry for dynamic client registration and change the current carbon server entry as adminService .